Zotero/Scenes/Master/Libraries/Views/LibrariesViewController.swift

```diff
@@ -207,3 +212,18 @@ extension LibrariesViewController: UITableViewDataSource, UITableViewDelegate {
         }
     }
 }
+
+extension LibrariesViewController: IdentifierLookupWebViewProvider {
+    func addWebView() -> WKWebView {
+        let webView = WKWebView()
+        webView.isHidden = true
+        view.insertSubview(webView, at: 0)
+        return webView
+    }
+    
+    func removeWebView(_ webView: WKWebView) {
+        if view.subviews.contains(where: { $0 == webView }) {
```

Comment:
Do you need to check this? Does this even need to be called here? Can't you just call `webView.removeFromSuperview()` directly?

Reply:
Check https://github.com/zotero/zotero-ios/pull/707/commits/c79834da3fe9b9acead80374f8016ff0327d007b